api/AudioContext.json

```diff
@@ -115,30 +129,70 @@
+              "version_added": "6.1",
               "prefix": "webkit"
             },
             "safari_ios": {
-              "version_added": true,
+              "version_added": "6.1",
               "prefix": "webkit"
             },
-            "samsunginternet_android": {
-              "version_added": "6.0",
-              "notes": [
-                "Each tab is limited to 6 audio contexts in Samsung Internet; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Samsung Internet'>Per-tab audio context limitation in Samsung Internet</a>.",
-                "If <code>latencyHint</code> isn't valid, Samsung Internet throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Samsung Internet'>Non-standard exceptions in Samsung Internet</a> for details."
-              ]
-            },
-            "webview_android": {
-              "version_added": "55"
-            }
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0",
+                "notes": [
+                  "Prior to Samsung Internet 9.0, each tab is limited to 6 audio contexts in Samsung Internet; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
+                  "If <code>latencyHint</code> isn't valid, Samsung Internet throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Chrome'>Non-standard exceptions in Chrome</a> for details."
+                ]
+              },
+              {
+                "version_added": "1.0",
+                "version_removed": "7.0",
+                "prefix": "webkit"
+              }
+            ],
+            "webview_android": [
+              {
+                "version_added": "37",
+                "notes": [
+                  "Prior to WebView 66, each tab is limited to 6 audio contexts in WebView; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
+                  "If <code>latencyHint</code> isn't valid, WebView throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Chrome'>Non-standard exceptions in Chrome</a> for details."
+                ]
+              },
+              {
+                "version_added": "≤37",
```

Comment:
Yep, the prefixed AP was likely available in some earlier versions, but figuring out which ones is not worth the effort.